File: Course_01/C++ Language/Vetores/media_pares.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int i, n;

    cout << "Quantos elementos vai ter o vetor? ";
    cin >> n;

    int v[n], soma(0), contePares(0);
    double media;

    for (i = 0; i < n; i++) {
        cout << "Digite um numero: ";
        cin >> v[i];
    }

    for (i = 0; i < n; i++) {
        if (v[i] % 2 == 0) {
            contePares++;
            soma += v[i];
        }
    }

    if (soma == 0) {
        cout << "NENHUM NUMERO PAR";
    }
    else {
        media = (double) soma / contePares;
        cout << fixed << setprecision(1) << "MEDIA DOS PARES = " << media;
    }

    return 0;
}

File: Course_01/Python/Vetores/media_pares.py
n = int(input("Quantos elementos vai ter o vetor? "))

vet = [0 for x in range(n)]
somaPares = contagemPares = 0
for i in range(0, n):
    vet[i] = float(input("Digite um número: "))
    if vet[i] % 2 == 0:
        somaPares += vet[i]
        contagemPares += 1
if contagemPares == 0:
    print("Nenhum número par")
else:
    print(f"Média dos pares = {(somaPares / contagemPares):.1f}")